internals/cli/cmd_help.go

```diff
@@ -214,6 +214,10 @@ var HelpCategories = []HelpCategory{{
 	Label:       "Notices",
 	Description: "manage notices and warnings",
 	Commands:    []string{"warnings", "okay", "notices", "notice", "notify"},
+}, {
+	Label:       "Identities",
+	Description: "manage user identities",
+	Commands:    []string{"identities", "identity", "add-identities", "update-identities", "remove-identities"},
```

Comment:
We were considering having a special case where when you typed `pebble help` it'd show just:

```
$ pebble help
...
Commands can be classified as follows:

         Run: run
...
     Notices: warnings, okay, notices, notice, notify
  Identities: identities --help
```

And then they would type `pebble identities --help` for more info.

However, that's an annoying special case, and I think it's simplest just to follow the existing pattern. Now that `replace-identities` isn't its own command (it's `update-identities --replace`) it's a bit shorter anyway. The list is not ridiculously long:

```
$ pebble help
...
Commands can be classified as follows:

         Run: run
...
     Notices: warnings, okay, notices, notice, notify
  Identities: identities, identity, add-identities, update-identities, remove-identities
```

Reply:
I would argue that we specially had come to a consensus on this. Please push forward on adding support for this, it can be in another PR, but I think it should happen as this was one of the nice things to come out of the spec process.

Reply:
Fair enough -- updated.